hey.go

```diff
@@ -211,8 +211,8 @@ func main() {
 	signal.Notify(c, os.Interrupt)
 	go func() {
 		<-c
-		w.Finish()
-		os.Exit(1)
+		fmt.Println("\nInterrupted, stopping workers.")
```

Comment:
Are you sure you don't want to use a logger with level "error" or "info" at least?

Reply:
Given the style of the rest of the file, at least fmt.Fprintf(os.stderr, ....)

Reply:
Done. (Although technically this is not an error, if user breaks the application, we just stop the workers and let it print the partial results). But for sake of consistency I agree.